### ISSUE TITLE
entities for each DSMR version

### ISSUE THREAD

**Issue by RenewedMe** [home-assistant/home-assistant.github.io#26824]
### Feedback

Hi,

My Smart meter was just upgraded to a Landis & Gym 360 which is compatible with DSMR 5.0. I had expected that a number of entities that were not available in my old smart meter would now become available (like power per phase) but they did not; I still only have the same entities that I had with my old meter. It therefore would useful to see in the documentation which entity would be available with which DSMR version.

### URL

https://www.home-assistant.io/integrations/dsmr/

### Version

2023.3.6

**Comment by frenck** [comment]
That is not specific to only dsmr version, it depends on meter, and even different meter models too. Additionally countries are added into the mix.

It tried to make available what it can get 🤷‍♂️

**Comment by RenewedMe** [comment]
I managed to get out to work. The ‘problem’ was that the integration by default makes a number of entities unavailable. When I tried manually to make them available at first I did not see any result so I presumed this did not work. After trying a second time and being a bit more patient it did work.
It might be an idea to mention in the documentation that the integration does not check what can be made available but instead opts to make a standard set unavailable and then leaves it to the use to try to see if these entities can be made available?
